osu.Game/Screens/Edit/Compose/Components/SelectionBoxScaleHandle.cs

```diff
@@ -100,13 +100,13 @@ private void adjustScaleFromAnchor(ref Vector2 scale)
             if ((originalAnchor & Anchor.y0) > 0) scale.Y = -scale.Y;
         }
 
-        private void applyScale(bool shouldLockAspectRatio)
+        private void applyScale(bool shouldLockAspectRatio, bool ignoreAnchor = false)
```

Comment:
Not a huge fan of this new parameter's name but I'm drawing a blank as to what would be a better one. `useDefaultOrigin` maybe?